crates/next-core/src/next_font_google/mod.rs

```diff
@@ -72,9 +73,10 @@ impl ImportMappingReplacement for NextFontGoogleReplacer {
         let query = &*query_vc.await?;
         let options = font_options_from_query_map(*query_vc);
         let properties = get_font_css_properties(options).await?;
+        let request_id = *get_request_id(*query_vc).await?.await?;
         let js_asset = VirtualAssetVc::new(
                 attached_next_js_package_path(self.project_path)
-                    .join("internal/font/google/inter.js"),
+                    .join(&format!("internal/font/google/{:x?}.js", request_id)),
```

Comment:
would be nice to have the font name in there, but 🤷

Reply:
yeah, I thought the same... should be somewhat straightforward. I'll give it a go.